src/components/dashjs-api-control/dashjs-api-control.tsx

```diff
@@ -264,23 +276,23 @@ export class DashjsApiControl {
       <Host>
         <ion-accordion titleText="API">
           <div slot="title" style={{ display: 'flex', alignItems: 'center', alignSelf: 'flex-end' }}>
-            Auto start <ion-toggle id="autol" onIonChange={event => this.updateInformation(0, event)} checked={getStringLocally('api_autostart') === null ? false : getStringLocally('api_autostart') === 'true'}></ion-toggle>
+            Auto start <ion-toggle id="autol" onIonChange={event => this.updateAutostart(event)} checked={getStringLocally('api_autostart') === null ? false : getStringLocally('api_autostart') === 'true'}></ion-toggle>
           </div>
           <ion-grid>
             <ion-row>
               <ion-col size="2">
                 <ion-button shape="round" onClick={ev => this.presentPopover(ev)} class="fill_width">
-                  Select Stream<ion-icon name="arrow-dropdown"></ion-icon>
+                  Select Stream&nbsp;<ion-icon name="caret-down-circle"></ion-icon>
```

Comment:
you can use `slot="end"` instead of the `&nbsp;` hack for the icon https://ionicframework.com/docs/api/button